ui/widgets/nuxeo-html-editor.js

```diff
@@ -39,8 +40,77 @@ import './quill/quill-snow.js';
           div#editor > * {
             margin-top: 0;
           }
+
+          iron-icon {
+            height: 18px;
+            color: #444;
+          }
         </style>
 
+        <nuxeo-document-picker
+          id="picker"
+          provider="document_picker"
+          page-size="40"
+          schemas="dublincore,file"
+          enrichers="thumbnail,permissions,highlight"
+          search-name="document_picker"
+          on-picked="_onPickerSelected"
+        ></nuxeo-document-picker>
+
+        <div id="toolbar">
+          <span class="ql-formats">
+            <select class="ql-header">
+              <option value="1"></option>
+              <option value="2"></option>
+              <option value="3"></option>
+              <option value="4"></option>
+              <option value="5"></option>
+              <option value="6"></option>
+              <option selected></option>
+            </select>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-bold"></button>
+            <button class="ql-italic"></button>
+            <button class="ql-underline"></button>
+            <button class="ql-strike"></button>
```

Comment:
we are adding a bunch of buttons here without any inner text or aria-labels which ultimately will add a11y debt to this project. Maybe we can create a ticket later to address this issue.

Reply:
I think this deserves a dedicated ticket, as this is dept was already here before and we will probably have to come up with new labels.

Reply:
Exactly, these buttons were there before. If anything this represents an opportunity to make them accessible since we now fully control the Quill editor toolbar.
[ELEMENTS-1345: Quill editor toolbar options should be made accessible](https://jira.nuxeo.com/browse/ELEMENTS-1345)

Reply:
thanks for the ticket, Nixo 🙇‍♂️